### PR TITLE
[CC][master] Update from resource to operation level policy overview. 

### PR DESCRIPTION
**Pull request by suksw**
## Purpose
Update the following for Choreo Connect related docs
- Add API policy overview.
- Update menu for mediation. 
- Remove sections that says cc only supports policies at resource level.
- Update custom policy docs.

### CODE REVIEW

**Comment by renuka-fernando** [comment]
Let's update applicable flows in this doc https://apim.docs.wso2.com/en/latest/design/api-policies/choreo-connect-policies/create-custom-cc-policy/

File: en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Choreo Connect API Policies
+
+Choreo Connect supports API policies at the HTTP method level for request and response flows. Follow the steps given below to add policies to an API.
+
+1. Open the API Manager Publisher Portal.
+2. Select the API.
+3. Navigate to the **Policies** tab.
+4. Select **Choreo Connect** as the gateway type. 
+5. Then, drag and drop the policies to the relevant flow in the required order.
+
```
Comment (slahirucd7) [inline en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md:10]: Shall we mention about the requirement to click save button also ?

File: en/docs/design/api-policies/choreo-connect-policies/create-custom-cc-policy.md [hunk 2]
```diff
@@ -17,17 +30,17 @@ Choreo Connect supports the following operations and the policies supporting the
             <li>headerName</li>
             <li>headerValue</li>
         </ul></td>
-    <td>Request</td>
-    <td>Set a header in the request flow.</td>
+    <td>Request Response</td>
+    <td>Set a header in the request flow or response flow.</td>
```
Comment (slahirucd7) [inline en/docs/design/api-policies/choreo-connect-policies/create-custom-cc-policy.md:34]: Can we change this to `*Sets* a header in the request flow or response flow` ? If possible let's apply the verbs used in other rows also.

File: en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Choreo Connect API Policies
+
+Choreo Connect supports API policies at the HTTP method level for request and response flows. Follow the steps given below to add policies to an API.
+
+1. Open the API Manager Publisher Portal.
+2. Select the API.
+3. Navigate to the **Policies** tab.
```
Comment (slahirucd7) [inline en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md:7]: Can we change step 3 to -> `Navigate to the  Policies page relevant to the API`
In global level also there's a Policies page/tab.

File: en/docs/design/api-policies/attach-policy.md [hunk 4]
```diff
@@ -18,14 +18,14 @@ Follow the instructions below to attach one or more default policies that are sh
...
+        | Gateway         | Supported Flows         |
         |-----------------|-------------------------|
-        | Regular Gateway | **Operation level** granularity |
-        |                 | Supported flows under each operation: `Request`, `Response`, `Fault` |
-        | Choreo Connect  | **Resource level** granularity |
-        |                 | Supported flows under each resource: `Request`, `Response` |
+        | Regular Gateway | `Request`, `Response`, `Fault` |
+        | Choreo Connect  | `Request`, `Response` |
+
+        * Choreo Connect supports policies at **Operation level** 1.2.0 version onwards, while the older version Choreo Connect 1.1.0 only supports at resource level.
```
Comment (samgnaniah) [inline en/docs/design/api-policies/attach-policy.md:28]:
```suggestion
        * Choreo Connect supports policies at **Operation level** 1.2.0 version onwards, while the older version Choreo Connect 1.1.0 only supports policies at resource level.
```

File: en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+4. Select **Choreo Connect** as the gateway type.
```
Comment (samgnaniah) [inline en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md:8]:
```suggestion
4. Select **Choreo Connect** as the Gateway type. 
```

**Review by samgnaniah — APPROVED** [review]

**Review by renuka-fernando — APPROVED** [review]

File: en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md (hunk 3 above)
Comment (suksw) [inline en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md:7]:
```suggestion
3. Click **Policies** under **API Configurations** in the left menu.
```

File: en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Choreo Connect API Policies
+
+Choreo Connect supports API policies at the HTTP method level for request and response flows. Follow the steps given below to add policies to an API.
+
+1. Open the API Manager Publisher Portal.
+2. Select the API.
+3. Click **Policies** under **API Configurations** in the left menu.
+4. Select **Choreo Connect** as the Gateway type. 
+5. Then, drag and drop the policies to the relevant flow in the required order.
```
Comment (suksw) [inline en/docs/design/api-policies/choreo-connect-policies/cc-op-policy-overview.md:9]:
```suggestion
5. Then, drag and drop the policies to the relevant flow in the required order.
6. Scroll to the bottom of the page, click the arrow next to **Save**, and click **Save and deploy**.
```

**Pull request merged**